test/old/testdir/test_ins_complete.vim

```diff
@@ -2664,6 +2664,12 @@ func Test_complete_fuzzy_match()
   call feedkeys("A\<C-X>\<C-N>\<Esc>0", 'tx!')
   call assert_equal('hello help hero h', getline('.'))
 
+  set completeopt=fuzzy,menuone,menu,noselect
+  call setline(1, ['Text', 'ToText', ''])
+  call cursor(2, 1)
```

Comment:
It should be 3 but I made a mistake. I will fix it in follow up PR. Then port into #30189.